Mappers/HttpMapper.php

```diff
@@ -27,128 +36,152 @@
      */
     private $httpMapperConfig = null;
 
+    /**
+     * @var string
+     */
+    protected static $contentType = self::CONTENT_TYPE_JSON;
+
+    /**
+     * HttpMapper constructor.
+     * @param array $httpMapperConfig
+     * @param HttpClient $httpClient
+     */
     public function __construct(array $httpMapperConfig, HttpClient $httpClient)
     {
-        $this->setHttpMapperConfig($httpMapperConfig)->setHttpClient($httpClient);
+        $this->httpClient = $httpClient;
+        $this->httpMapperConfig = $httpMapperConfig;
     }
 
     /**
-     * @param $conditions
-     * @return Collection
-     * @throws EnergonException
+     * @param array $objectData
+     * @return ArraySerializable
+     * @throws HttpResourceNotFoundException
+     * @throws InfrastructureException
+     * @throws ValidationException
+     * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    public function load($conditions)
+    public function create(array $objectData): ArraySerializable
     {
-        return $this->sendHttpRequest('GET', $this->getHttpMapperConfig(self::LOAD_URL), ['query' => $conditions]);
+        return $this->sendHttpRequest('POST', $this->createUrl(), [], $objectData);
```

Comment:
Too many arguments, use class instead

Reply:
Please create also constant name for POST,GET, etc.
Agree we need Request object

Reply:
constant name for POST,GET, etc. - fixed
Agree we need Request object - don't agree